### DISCUSSION TITLE
Always tells me RuntimeError: Port is not idle

### DISCUSSION THREAD

**Discussion by Roy-Kid** [microsoft/nni#5605] [Q&A]
Hi, when I run nni, it always tells me port is not idle. How can I fix this?

The nni is running locally on WSL2 and not behind proxy. I have tried many ports not none of them work. I think the firewall is not the issue because my proxy can redirect requests from WSL to windows. How can I fix the problem?

**Comment by Nafees-060** [comment]
Run this  ` nnictl stop --all` first and then try.

**Comment by XinY-Z** [comment]
Or try this `pkill -f "<whatever port you used, usually it's 8080>"`

**Comment by cizhouyu** [comment]
Maybe you can check if you are running the correct command. I had configured 'experiment.config.trial_command' which called the file itself, hence the issue.
